inspirehep/migrator/tasks.py

```diff
@@ -183,8 +180,8 @@ def populate_mirror_from_file(source):
         print(f"Inserted {inserted_records} records into mirror")
 
 
-@shared_task(ignore_result=True)
-def continuous_migration():
+@shared_task(ignore_results=False)
+def continuous_migration(max_timeout=60 * 60, force_wait=False):
```

Comment:
why do you think these two parameters are needed? why 1h? I think you can just fire and forget these tasks, no need to wait.

Reply:
If some weird thing will happen, like worker crash etc. this will be waiting forever, and will block migrations if there won't be any timeout.

Reply:
why will it block? if you're not waiting, you're releasing the lock. If a worker crashes during a task, it's true that we might get inconsistent data. What about setting `acks_late=True` (see https://docs.celeryproject.org/en/latest/reference/celery.app.task.html#celery.app.task.Task.acks_late) to redo the task in case the worker crashes? all of these tasks should be safe to perform them several times.